apps/spruce/src/constants/queryParams.ts

```diff
@@ -0,0 +1,4 @@
+export enum PaginationQueryParams {
+  Page = "page",
+  Limit = "limit",
+}
```

Comment:
These values are already defined in `src/components/Table/utils.ts`. Could we reuse those?